app/components/atoms/social-media-buttons/social-media-button.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import {ComponentMeta, ComponentStory} from '@storybook/react';
+
+import {
+  InstagramButton,
+  FacebookButton,
+  TwitterButton,
+} from './social-media-button.component';
+
+export default {
+  title: 'Atoms/SocialMediaButton',
+  component: InstagramButton,
+} as ComponentMeta<typeof InstagramButton>;
+
+const Template: ComponentStory<typeof InstagramButton> = () => (
```

Comment:
same suggestions apply from https://github.com/tepachelabs/perro-cafe-storefront/pull/33 apply for this.